integration/samples/externals/specs/umd.ts

```diff
@@ -38,16 +35,58 @@ describe(`@sample/externals`, () => {
     });
 
     it(`should import '@angular/core'`, () => {
-      expect(BUNDLE).to.contain("require('@angular/core')")
+      expect(BUNDLE).to.contain("require('@angular/core')");
+    });
+
+    it(`should import 'rxjs/Observable`, () => {
+      expect(BUNDLE).to.contain("require('rxjs/Observable')");
+    });
+
+    it(`should not import 'lodash.template`, () => {
+      expect(BUNDLE).to.not.contain("require('lodash.template')");
+    });
+
+    it(`should not import transitive dependency 'lodash.template`, () => {
+      expect(BUNDLE).to.not.contain("require('lodash.templatesettings')");
     });
 
-    it(`should embed 'createCommonjsModule' method`, () => {
-      expect(BUNDLE).to.contain("function createCommonjsModule")
+    it(`should embed transitive 'templateSettings' object`, () => {
+      expect(BUNDLE).to.contain('var templateSettings = {');
     });
 
-    it(`should embed 'fn.start' method`, () => {
-      expect(BUNDLE).to.contain("n.start = function")
+    it(`should embed 'template' function`, () => {
+      expect(BUNDLE).to.contain('template(string, options, guard)');
     });
   });
+});
 
+describe(`@sample/externals/testing`, () => {
+  describe(`UMD Bundle`, () => {
+    let BUNDLE;
+    before(() => {
+      BUNDLE = fs.readFileSync(path.resolve(__dirname, '../dist/bundles/sample-externals-testing.umd.js'), {
+        encoding: 'utf-8'
+      });
+    });
+
+    it(`should import '@sample/externals'`, () => {
+      expect(BUNDLE).to.contain("require('@sample/externals')");
+    });
+
+    it(`should not import 'lodash.template`, () => {
+      expect(BUNDLE).to.not.contain("require('lodash.template')");
+    });
+
+    it(`should not import transitive dependency 'lodash.template`, () => {
+      expect(BUNDLE).to.not.contain("require('lodash.templatesettings')");
+    });
+
+    it(`should embed transitive 'templateSettings' object`, () => {
+      expect(BUNDLE).to.contain('var templateSettings = {');
+    });
+
+    it(`should embed 'template' function`, () => {
+      expect(BUNDLE).to.contain('template(string, options, guard)');
+    });
```

Comment:
Note: test specs for a bundled depency (prior "embedded")

Reply:
bundled depencencies are embedded in UMD bundles. This test verifies the behaviour.